<launch>
  <group ns="move_head2">
  <node pkg="annotated_map_builder" type="wait_for_k_messages_action.py" output="screen" name="wait_k_messages_action">
    <remap from="topic" to="/stereo/raw_stereo_throttled"/>
    <param name="count" value="3" />
  </node>

  <node pkg="annotated_map_builder" type="move_head_action.py" name="move_head_action" output="screen">
    <param name="wait_action" value="/move_head2/wait_k_messages_action"/>  
    <param name="head_configs" value="[[0.0,0.3]]" />
    
  </node>
  </group>
</launch>
